os/drivers/audio/audio_null.c

```diff
@@ -181,7 +218,7 @@ static const struct audio_ops_s g_audioops = {
 static int null_getcaps(FAR struct audio_lowerhalf_s *dev, int type, FAR struct audio_caps_s *caps)
 {
 	audvdbg("type=%d\n", type);
-
+	
```

Comment:
It seems this is an unnecessary Indentation.

Reply:
fixed.